### ISSUE TITLE
FormatTime broken with v1.5.2+ when TZ is not set (Local)

### ISSUE THREAD

**Issue by SLoeuillet** [ClickHouse/clickhouse-go#535]
### Issue description
With 1.5.1, FormatTime didn't use timezone at all, inserting a time.Time like time.Now() worked even if TZ was not set (Local)
With 1.5.2+, FormatTime would write "Local" in the timestamp send to clickhouse that would "fail to load Local timezone"

### How v2 handles that :
```bind.go:134, method format 

        case time.Time:
                switch v.Location().String() {
                case "Local":
                        return fmt.Sprintf("toDateTime(%d)", v.Unix())
                case tz.String():
                        return v.Format("toDateTime('2006-01-02 15:04:05')")
                }
                return v.Format("toDateTime('2006-01-02 15:04:05', '" + v.Location().String() + "')")
```
### How v1.5.2 handles that :

```helpers.go:142

func formatTime(v time.Time) string {
        return v.Format("toDateTime('2006-01-02 15:04:05', '" + v.Location().String() + "')")
}
```

**Comment by SLoeuillet** [comment]
Something like 
```
diff --git a/helpers.go b/helpers.go
index 7e10d06..782c401 100644
--- a/helpers.go
+++ b/helpers.go
@@ -140,5 +140,11 @@ func quote(v driver.Value) string {
 }
 
 func formatTime(v time.Time) string {
+       switch v.Location().String() {
+       case "Local":
+               return fmt.Sprintf("toDateTime(%d)", v.Unix())
+       case tz.String():
+               return v.Format("toDateTime('2006-01-02 15:04:05')")
+       }
        return v.Format("toDateTime('2006-01-02 15:04:05', '" + v.Location().String() + "')")
 }
```

**Comment by SLoeuillet** [comment]
1.5.2+ gives that error with no TZ set :
failed to query CH: code: 1000, message: Exception: Cannot load time zone Local

**Comment by gingerwizard** [comment]
Closed via #536
